### DISCUSSION TITLE
一个不那么辣眼的应用图标设计

### DISCUSSION THREAD

**Discussion by kwaa** [SagerNet/SagerNet#173] [Show and tell]
所以我又[重新发了一遍](https://github.com/SagerNet/SagerNet/issues/104#issuecomment-860254763)。

![icon](https://user-images.githubusercontent.com/50108258/121818599-ba820a80-ccba-11eb-9125-12e0cda77f97.png)

**Comment by nekohasekai** [comment]
丑...

**Reply by kwaa** [reply]
🤔那要看跟什么比了... 至少配色比较舒服 图标我确实没什么好想法
但 outline vpn_key 不会一眼认成 Shadowsocks

**Reply by wizos** [reply]
> 🤔那要看跟什么比了... 至少配色比较舒服 图标我确实没什么好想法
> 但 outline vpn_key 不会一眼认成 Shadowsocks

配色舒服？我不是杠哈，但这土黄色花纹，配上了金黄色的底，你是认真的吗😂

**Reply by kwaa** [reply]
> > 🤔那要看跟什么比了... 至少配色比较舒服 图标我确实没什么好想法
> > 但 outline vpn_key 不会一眼认成 Shadowsocks
> 
> 配色舒服？我不是杠哈，但这土黄色花纹，配上了金黄色的底，你是认真的吗😂

金黄 × 
Material Amber √（翻译过来是叫琥珀色？）
线条我做的时候偷懒了，应该用深一点的主色而不是正片叠底黑
但这只是个背景而已。。。配合白色图标我觉得对观感影响不大（除非你喜欢对比色背景那另说）

**Comment by kwaa** [comment]
样式2

![preview](https://user-images.githubusercontent.com/50108258/125650730-1e7e8b65-3353-460f-be09-2a615d43b41a.png)

**Reply by KagurazakaNyaa** [reply]
比上面那个好看多了

**Reply by kwaa** [reply]
> 比上面那个好看多了

🤔这个我放弃了符合应用名&接近原背景色，变成「好看就行」
所以比较能自由发挥

**Reply by KagurazakaNyaa** [reply]
> > 比上面那个好看多了
> 
> 🤔这个我放弃了符合应用名&接近原背景色，变成「好看就行」
> 所以比较能自由发挥

才发现你上面那个有个耳朵（笑

**Comment by U707T** [comment]
我觉得以前AX图标就挺好看的，希望能按照原来那个差不多来整
